modular_skyrat/master_files/code/modules/language/language_holder.dm

```diff
@@ -54,3 +54,62 @@ GLOBAL_DATUM_INIT(language_holder_adjustor, /datum/language_holder_adjustor, new
+								/datum/language/uncommon = list(LANGUAGE_ATOM),
+								/datum/language/machine = list(LANGUAGE_ATOM),
+								/datum/language/draconic = list(LANGUAGE_ATOM),
+								/datum/language/moffic = list(LANGUAGE_ATOM),
+								/datum/language/calcic = list(LANGUAGE_ATOM),
+								/datum/language/voltaic = list(LANGUAGE_ATOM),
+								/datum/language/nekomimetic = list(LANGUAGE_ATOM),
+								/datum/language/gutter = list(LANGUAGE_ATOM),
+								/datum/language/panslavic = list(LANGUAGE_ATOM),
+								/datum/language/skrell = list(LANGUAGE_ATOM),
+								/datum/language/spacer = list(LANGUAGE_ATOM),
+								/datum/language/xerxian = list(LANGUAGE_ATOM),
+								/datum/language/vox = list(LANGUAGE_ATOM),
+								/datum/language/yangyu = list(LANGUAGE_ATOM),
+								/datum/language/schechi = list(LANGUAGE_ATOM),
+								/datum/language/monkey = list(LANGUAGE_ATOM),
+								/datum/language/slime = list(LANGUAGE_ATOM),
+								/datum/language/beachbum = list(LANGUAGE_ATOM),
+								/datum/language/mushroom = list(LANGUAGE_ATOM),
+								/datum/language/shadowtongue = list(LANGUAGE_ATOM),
+								/datum/language/buzzwords = list(LANGUAGE_ATOM),
+								/datum/language/terrum = list(LANGUAGE_ATOM),
+								/datum/language/sylvan = list(LANGUAGE_ATOM),
+								/datum/language/siiktajr = list(LANGUAGE_ATOM),
+								/datum/language/canilunzt = list(LANGUAGE_ATOM),
+								/datum/language/akulan = list(LANGUAGE_ATOM)
+								)
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/uncommon = list(LANGUAGE_ATOM),
+							/datum/language/machine = list(LANGUAGE_ATOM),
+							/datum/language/draconic = list(LANGUAGE_ATOM),
+							/datum/language/moffic = list(LANGUAGE_ATOM),
+							/datum/language/calcic = list(LANGUAGE_ATOM),
+							/datum/language/voltaic = list(LANGUAGE_ATOM),
+							/datum/language/nekomimetic = list(LANGUAGE_ATOM),
+							/datum/language/gutter = list(LANGUAGE_ATOM),
+							/datum/language/panslavic = list(LANGUAGE_ATOM),
+							/datum/language/skrell = list(LANGUAGE_ATOM),
+							/datum/language/spacer = list(LANGUAGE_ATOM),
+							/datum/language/xerxian = list(LANGUAGE_ATOM),
+							/datum/language/vox = list(LANGUAGE_ATOM),
+							/datum/language/yangyu = list(LANGUAGE_ATOM),
+							/datum/language/schechi = list(LANGUAGE_ATOM),
+							/datum/language/monkey = list(LANGUAGE_ATOM),
+							/datum/language/slime = list(LANGUAGE_ATOM),
+							/datum/language/beachbum = list(LANGUAGE_ATOM),
+							/datum/language/mushroom = list(LANGUAGE_ATOM),
+							/datum/language/shadowtongue = list(LANGUAGE_ATOM),
+							/datum/language/buzzwords = list(LANGUAGE_ATOM),
+							/datum/language/terrum = list(LANGUAGE_ATOM),
+							/datum/language/sylvan = list(LANGUAGE_ATOM),
+							/datum/language/siiktajr = list(LANGUAGE_ATOM),
+							/datum/language/canilunzt = list(LANGUAGE_ATOM),
+							/datum/language/akulan = list(LANGUAGE_ATOM)
+							)
```

Comment:
Please format these lists properly, so they aren't so much of an eyesore, and incredibly indented for no reason.

Reply:
Fixed it myself.